infection.json.dist

```diff
@@ -10,5 +10,12 @@
         "badge": {
             "branch": "master"
         }
+    },
+    "mutators": {
+        "MethodCallRemoval": {
+            "ignore": [
+                "Infection\\Finder\\SourceFilesFinder::__construct::63"
```

Comment:
This is for parent's `__construct`. It does some things, but they're quite hard to test, and that is kind of out of scope of our project.

Reply:
That's the reason I put it in there. Granted, I do have to disagree to some point. It is quite easy to test. I had already commented the test code in #660. The problem is that you have to test using reflection, something most people tend to frown upon.

Reply:
Yeah, true. What do you think about adding this rationale to the PR description? It is not immediately obvious, that why I had to comment.